modules/nf-core/vcflib/vcffilter/meta.yml

```
@@ -0,0 +1,48 @@
+name: "vcflib_vcffilter"
+description: write your description here
+keywords:
+  - filter
+  - variant
+  - vcf
+  - quality
+tools:
+  - "vcflib":
+      description: "Command line tools for parsing and manipulating VCF files."
+      homepage: "https://github.com/vcflib/vcflib"
+      documentation: "https://github.com/vcflib/vcflib"
+      tool_dev_url: "https://github.com/vcflib/vcflib"
+      doi: "10.1371/journal.pcbi.1009123"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test_sample_1' ]
+  - vcf:
+      type: file
+      description: VCF file
+      pattern: "*.{vcf,vcf.gz}"
+  - tbi:
+      type: file
+      description: Index file
+      pattern: "*.{tbi}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - vcf:
+      type: file
+      description: Filtered VCF file
+      pattern: "*.{vcf,vcf.gz}"
```

```suggestion
      pattern: "*.{vcf.gz}"
```

Since you gzip the output